src/promptflow-devkit/tests/sdk_cli_test/e2etests/test_cli.py

```diff
@@ -1311,6 +1311,36 @@ def get_node_settings(_flow_dag_path: Path):
         finally:
             shutil.rmtree(output_path, ignore_errors=True)
 
+    def test_flex_flow_build(self):
+        from promptflow._cli._pf.entry import main
+
+        with tempfile.TemporaryDirectory() as temp:
+            temp = Path(temp)
+            cmd = (
+                "pf",
+                "flow",
+                "build",
+                "--source",
+                f"{EAGER_FLOWS_DIR}/chat-basic/flow.flex.yaml",
+                "--output",
+                temp.as_posix(),
+                "--format",
+                "docker",
+            )
+            sys.argv = list(cmd)
+            main()
+            assert (temp / "connections").is_dir()
+            assert (temp / "flow").is_dir()
+            assert (temp / "runit").is_dir()
+            assert (temp / "Dockerfile").is_file()
```

Comment:
Check if connection (if used in flex flow) included in Dockerfile?

Reply:
Of course, I have already added it.